llvm/docs/HowToAddABuilder.rst

```diff
@@ -179,6 +179,74 @@ Here are the steps you can follow to do so:
    buildbot.tac file to change the port number from 9994 to 9990 and start it
    again.
 
+Testing a Builder Config Locally
+================================
+
+It's possible to test a builder running against a local version of LLVM's
+buildmaster configuration. This can be helpful to allow quickly identifying
+and iterating over fixes to any issues in either the changes that introduce
+the new builder, or the machine configuration for your worker (preinstalled
+packages etc). A buildmaster launched in this "local testing" mode will bind
```

Comment:
Since this sentence includes security advice, I would make it into bullet points like:
```
A buildmaster launced in this "local testing" mode will:
* Bind only to local interfaces
* Use SQLLite as the database
* Use a fixed (single?) password for all workers
* Disable extras like GitHub Authentication
```

Reply:
On the grounds that folks should read and (to themselves) agree with each point in turn.